### PR TITLE
Revert "SCP 294 balance"

### PR DESCRIPTION
**Pull request by boy2mantwicethefam**
Sneaky coder don't sneaky code

### CODE REVIEW

**Comment by ghost** [comment]
>1-12 ratio

**Comment by SonixApache** [comment]
fuck off retard

**Comment by boy2mantwicethefam** [comment]
t. banned shitter

**Comment by Durgen** [comment]
back in my day the thread paid attention to github and threw shitfits when this happened

**Comment by boy2mantwicethefam** [comment]
Now anons become angry when there are features being added or removed instead

**Comment by tuypo1** [comment]
>back in my day the thread paid attention to github and threw shitfits when this happened

I blame reddit for the lack of attention paid to the github

**Comment by SonixApache** [comment]
shitfits are reserved for things that actually matter, like dylan tier balance changes

File: code/game/machinery/scp_294.dm [hunk 1]
```diff
@@ -18,8 +18,7 @@
 	dispensable_reagents = null
 	var/list/prohibited_reagents = list(ADMINORDRAZINE)
 
-	machine_flags = FIXED2WORK
-	mech_flags = MECH_SCAN_FAIL
```
Comment (dsfartgeg) [inline code/game/machinery/scp_294.dm:22]: Why'd you get rid of `MECH_SCAN_FAIL`?  Trying to sneak one past?

Comment (ghost) [reply code/game/machinery/scp_294.dm:22]: I'm not entirely sure whether this is sarcasm but this is what it was before.

Comment (boy2mantwicethefam) [reply code/game/machinery/scp_294.dm:22]: it was part of the reverted PR, made scans fail

**Comment by Durgen** [comment]
if this is so unimportant and innocuous then surely it will get merged

**Comment by SonixApache** [comment]
it already was

**Comment by ShiftyRail** [comment]
I would personally try to do something that is more "consensual"
Try to make them still moveable but not scannable.

**Pull request closed without merging**